Formula/drogon.rb

```diff
@@ -1,39 +1,29 @@
-# Documentation: https://docs.brew.sh/Formula-Cookbook
-#                https://rubydoc.brew.sh/Formula
-# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
 class Drogon < Formula
   desc "HTTP(S) Web Application Framework (C++14/17 based)"
   homepage "https://drogon.docsforge.com"
-  license "MIT"
   url "https://github.com/drogonframework/drogon/archive/refs/tags/v1.7.2.tar.gz"
   sha256 "a682fafc3619ba69d3d9278a80f6a386d1effcc3692b47e976de7cf9ff89d6b8"
+  license "MIT"
   head "https://github.com/drogonframework/drogon.git"
 
-  resource "trantor" do
-    url "https://github.com/an-tao/trantor.git",
-        revision: "bef06c4f5e2dfc80303aee06a12770d09d5e8f5b"
-  end
-
   depends_on "cmake" => :build
+  depends_on "brotli"
+  depends_on "c-ares"
   depends_on "jsoncpp"
-  depends_on "ossp-uuid"
   depends_on "openssl@1.1"
+  depends_on "ossp-uuid"
   depends_on "zlib"
-  depends_on "c-ares"
-  depends_on "brotli"
+
+  resource "trantor" do
+    url "https://github.com/an-tao/trantor.git",
+        revision: "bef06c4f5e2dfc80303aee06a12770d09d5e8f5b"
+  end
 
   def install
-    # ENV.deparallelize  # if your formula fails when building in parallel
-    # Remove unrecognized options if warned by configure
-    # https://rubydoc.brew.sh/Formula.html#std_configure_args-instance_method
-    # system "./configure", *std_configure_args, "--disable-silent-rules"
-    # head gets trantor as a git submodule
```

Comment:
Please add the complete comment with parallelization information back (basically revert this hunk).

Reply:
The comment was also pointed out by `brew audit --strict Formula/drogon.rb`, but do you still need it (just to confirm)?

```console
$ brew audit --strict Formula/drogon.rb
...
drogon:
  * 23: col 5: Please remove default template comments
  * 24: col 5: Please remove default template comments
```

Reply:
You can replace the default comment with:

# TODO: Verify Parallelization - https://github.com/drogonframework/homebrew-drogon/issues/4

But please leave the last line in, as it is not part of the default comment.